src/OpenTelemetry.Extensions.Hosting/OpenTelemetryBuilder.cs

```diff
@@ -76,9 +77,16 @@ internal OpenTelemetryBuilder(IServiceCollection services)
     /// Adds metric services into the builder.
     /// </summary>
     /// <remarks>
-    /// Note: This is safe to be called multiple times and by library authors.
+    /// Notes:
+    /// <list type="bullet">
+    /// <item>This is safe to be called multiple times and by library authors.
     /// Only a single <see cref="MeterProvider"/> will be created for a given
-    /// <see cref="IServiceCollection"/>.
+    /// <see cref="IServiceCollection"/>.</item>
+    /// <item>This method automatically calls <see
```

Comment:
This might not be helpful enough for users. Maybe we could add a simple note as well saying something like this method would respect the appSettings.json configuration as well.

Reply:
Add these remarks to the other overload method as well.

Reply:
* I don't really want to call out `appSettings.json` directly because it is really `IConfiguration` which could be lots of other things too. There is also a programmatic version: `services.AddMetrics(builder => builder.EnableMetrics("MyMeterName"))`. I think what may be best is we link to some documentation here. I checked with @samsp-msft, I don't think that documentation exists yet. Let's circle back to this later?

* The overload is getting the same remarks via `inheritdoc`: `/// <remarks><inheritdoc cref="WithMetrics()" path="/remarks"/></remarks>`